docs/api/hardhat/hardhat-zksync-deploy.md

```diff
@@ -1,18 +1,26 @@
 # `hardhat-zksync-deploy`
 
-This plugin provides utilities for deploying smart contracts on zkSync with artifacts built by the `@matterlabs/hardhat-zksync-solc` or `@matterlabs/hardhat-zksync-vyper` plugins.
+This plugin provides utilities for deploying smart contracts on zkSync Era with artifacts built by the official `@matterlabs/hardhat-zksync-solc` plugins.
 
 ::: warning
+- Contracts must be compiled using the official `@matterlabs/hardhat-zksync-solc` plugin. 
+- Contracts compiled with other compilers will fail to deploy to zkSync Era using this plugin.
+:::
 
```

Comment:
Why are you removing the vyper reference here?

Reply:
I removed it, to put it as a warning in the Vyper section.

Reply:
this is another duplicate content problem ... why not add this to a main page like under here instead:

https://era.zksync.io/docs/api/hardhat/getting-started.html#prerequisites

Reply:
@kailash-manasarovar updated.